### ISSUE TITLE
`app` project architecture is not documented anywhere

### ISSUE THREAD

**Issue by graywolf** [hanami/hanami#845]
When I want to generate new hanami application, I can select between `container` and `app` project architecture. However, these are not documented anywhere I could find. http://hanamirb.org/guides/1.0/command-line/applications/ is completely silent on that matter.

Would be nice to have documented what the difference is and when to use which one.

**Comment by cllns** [comment]
Sorry for the confusion! We used to support `container` and `app` architectures, before `1.0` was released. We decided to simplify things and only support the `container` architecture. We now refer to as a Hanami "project", and don't talk about any particular architecture in this sense anymore. 

Container is `apps/web`, `apps/api`, `apps/admin` etc. The `app` architecture just has one, in side of an `app/` folder. We hope that `app` architecture can come back via a plugin at some point 😄 

Our release for `1.0` accidentally retained the command-line option. Trying to use `--architecture=app` fails, since files are missing.

This will be fixed in `1.1.0` (available as `1.1.0.beta3` right now). See #815 for the work done to remove references to those architecture (on the `develop` branch, which is that next minor release).
